test/matrix.cc

```diff
@@ -37,6 +37,7 @@ class SimpleCost final : public DynamicCost {
   }
 
   bool Allowed(const DirectedEdge* edge,
+               const bool /*is_dest*/,
```

Comment:
in some cases you skip `/*is_dest*/` but in some you add it. is it on purpose ?

Reply:
I saw it in code below https://github.com/valhalla/valhalla/blob/0b0300b97096aec7b519d6b32dbb3fee0734ce95/test/matrix.cc#L45 and did the same. Where do I skip? This comments need for readable code, also it suppresses "unused variable" warning.  If a var be used, we can uncomment a var name. It is convenient.

Reply:
I see, you followed local code style. ok